src/frontend/admin/src/components/templates/organizations/form/sections/OrganizationAddressForm.tsx

```diff
@@ -0,0 +1,147 @@
+import * as React from "react";
+import * as Yup from "yup";
+import { useForm } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import Box from "@mui/material/Box";
+import Grid from "@mui/material/Unstable_Grid2";
+import { FormattedMessage, useIntl } from "react-intl";
+import Typography from "@mui/material/Typography";
+import {
+  DTOOrganizationAddress,
+  Organization,
+} from "@/services/api/models/Organization";
+import { SimpleCard } from "@/components/presentational/card/SimpleCard";
+import { RHFProvider } from "@/components/presentational/hook-form/RHFProvider";
+import { RHFTextField } from "@/components/presentational/hook-form/RHFTextField";
+import { useOrganizations } from "@/hooks/useOrganizations/useOrganizations";
+import { organizationAddressFormMessages } from "@/components/templates/organizations/form/translations";
+import { RHFSelect } from "@/components/presentational/hook-form/RHFSelect";
+
+type Props = {
+  organization: Organization;
+};
+export function OrganizationAddressForm({ organization }: Props) {
+  const intl = useIntl();
+  const organizationQuery = useOrganizations({}, { enabled: false });
+  const organizationAddress = organization?.addresses?.[0];
+
+  const getDefaultValues = () => {
+    return {
+      title: organizationAddress?.title ?? "",
+      country: organizationAddress?.country ?? "FR",
+      address: organizationAddress?.address ?? "",
+      postcode: organizationAddress?.postcode ?? "",
+      city: organizationAddress?.city ?? "",
+      first_name: organizationAddress?.first_name ?? "",
+      last_name: organizationAddress?.last_name ?? "",
+    };
+  };
+
+  const RegisterSchema = Yup.object().shape({
+    title: Yup.string().required(),
+    address: Yup.string().required(),
+    postcode: Yup.string().required(),
+    city: Yup.string().required(),
+    country: Yup.string().required(),
+    first_name: Yup.string().required(),
+    last_name: Yup.string().required(),
+  });
+
+  const methods = useForm({
+    resolver: yupResolver(RegisterSchema),
+    defaultValues: getDefaultValues() as any, // To not trigger type validation for default value
+  });
+
+  const onSubmit = async (payload: DTOOrganizationAddress) => {
+    if (!organizationAddress) {
+      await organizationQuery.methods.addAddress(organization.id, payload);
+    } else {
+      await organizationQuery.methods.updateAddress(
+        organization.id,
+        organizationAddress.id,
+        payload,
+      );
+    }
+  };
+
+  return (
```

Comment:
Why is there a separate form for the address? Is there a reason to not include it in the existing form?

Reply:
Because currently, there is a dedicated endpoint.